src/core_ocean/shared/mpas_ocn_vel_tidal_potential.F

```diff
@@ -137,15 +138,17 @@ subroutine ocn_vel_tidal_potential_tend(meshPool, forcingPool, ssh, tend, err)!{
 
       nEdges = nEdgesArray( 1 )
 
-      !$omp do schedule(runtime) private(cell1, cell2, invdcEdge, k)
+      !$omp do schedule(runtime) private(cell1, cell2, invdcEdge, potentialGrad, k)
       do iEdge=1,nEdges
          cell1 = cellsOnEdge(1,iEdge)
          cell2 = cellsOnEdge(2,iEdge)
          invdcEdge = 1.0_RKIND / dcEdge(iEdge)
 
+         potentialGrad = gravity * invdcEdge * ( tidalPotentialEta(cell2) - tidalPotentialEta(cell1) &
+                                 + config_self_attraction_and_loading_beta*(ssh(cell2) - ssh(cell1)))
+
          do k=1,maxLevelEdgeTop(iEdge)
-            tend(k,iEdge) = tend(k,iEdge) + gravity * edgeMask(k,iEdge) * invdcEdge * ( tidalPotentialEta(cell2) - tidalPotentialEta(cell1) &
-                                                                        + config_self_attraction_and_loading_beta*(ssh(cell2) - ssh(cell1)))
+            tend(k,iEdge) = tend(k,iEdge) -  edgeMask(k,iEdge) * potentialGrad 
```

Comment:
Good  catch

Reply:
Have a minus sign error here.

Reply:
Actually, for consistency with `ssh_gradient`, make the minus at line 147.

Reply:
Look at `src/core_ocean/shared/mpas_ocn_vel_pressure_grad.F` line 172 for contrast.

Reply:
Thanks for catching this. It's taken care of now.